lib/beaker-hostgenerator/cli.rb

```diff
@@ -116,69 +117,81 @@ def initialize(argv = ARGV.dup)
         opts.on('-h',
                 '--help',
                 'Display command help.') do
-          puts opts
-          exit
+          @options[:help] = opts.to_s
+        end
+
+        opts.on('-v',
+                '--version',
+                'Display beaker-hostgenerator version number.') do
+          @options[:version] = true
         end
       end
 
       optparse.parse!(argv)
+      @layout = argv[0]
+    end
 
-      if @options[:list_supported_values]
-        print_supported_values
-        raise BeakerHostGenerator::Exceptions::SafeEarlyExit
```

Comment:
@waynr We don't need the SafeEarlyExit anymore with these changes, so we could delete the class, but...

It's part of our API, and since we're holding off on breaking changes until 1.0 I left it defined.
I'll add a ticket to delete it to that Epic.